autorest/codegen/models/imports.py

```diff
@@ -52,14 +52,22 @@ def __hash__(self):
                 retval += hash(getattr(self, attr))
         return retval
 
+class TypeDefinition:
```

Comment:
can you add a changelog as well? thanks!

Reply:
Yes, added in the changelog file.